### DISCUSSION TITLE
Grafana tool in SOC console

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#9029] [Unsupported Versions]
Hello it's possibile to hide grafana tool in SOC console for only dedicated users?

**Comment by TotieBash** [comment]
Ow, after reading your question I realize the specifics of your question "only dedicated users".  Disregard my answer below.

Its in tools.json file.  Copy it from salt default to local and then vi edit the file.
sudo cp /opt/so/saltstack/default/salt/soc/files/soc/tools.json /opt/so/saltstack/local/salt/soc/files/soc/
see "links" ref: https://docs.securityonion.net/en/2.3/soc-customization.html?highlight=customization

**Reply by dgmyr** [reply]
I presume that modifying the tools.json file the modification will be applied to all users registred in Security Onion, it's possible to apply only for some users based on their roles?

Thanks
